app/services/imo_api_service.rb

```diff
@@ -28,22 +28,29 @@ def send_invitation(patient, phone_number: nil)
     phone = phone_number || patient.latest_mobile_number
     locale = patient.locale
     url = IMO_BASE_URL + "send_invite"
+
+    unless locale.in?(["bn-BD", "en"])
```

Comment:
I don't think you need this check here - Rails will raise an error by default if this path would somehow get called in a locale w/o the IMO translations.

Reply:
That wasn't my experience. In specs, it defaulted to English. I think this is the cause: 
```
    config.i18n.fallbacks = [:en]
    config.i18n.default_locale = :en
 ```